tests/js-custom-authorization/custom_authorization.py

```diff
@@ -657,6 +658,170 @@ def run_api(args):
         network = test_metrics_logging(network, args)
 
 
+def test_reused_interpreter_behaviour(network, args):
+    primary, _ = network.find_nodes()
```

Comment:
Turning on the caching for the full_js test may be a good way to check that this is working as intended too. Something else we could do is add to the stats with a number of runtime instantiations per endpoint, to be compared to calls.

Reply:
Agreed and done - this is actually a _huge_ perf speedup, so I'm trying the same on the non-full app to compare. If its a speedup there, we may drop perf-testing `full_js` in favour of testing the small app in `cached` and `uncached` modes.